app/survey_config/survey_config.py

```diff
@@ -46,11 +46,15 @@ class SurveyConfig:
     sign_out_button_text: str = lazy_gettext("Save and exit survey")
     contact_us_url: str = field(init=False)
     cookie_settings_url: str = field(init=False)
+    cookie_address: str = field(init=False)
     privacy_and_data_protection_url: str = field(init=False)
 
     def __post_init__(self):
         self.contact_us_url: str = f"{self.base_url}/contact-us/"
         self.cookie_settings_url: str = f"{self.base_url}/cookies/"
+        self.cookie_address: str = self.cookie_settings_url.split("://")[-1].split("/")[
```

Comment:
I think it would be better named as `cookie_domain`. I would also add a comment of what we are trying to do, such as get the FQDN of the cookie settings URL.

Reply:
Added!